src/utils/sort.js

```diff
@@ -26,13 +26,13 @@ export const sort = (type, direction) => {
   case 'basic':
   default:
     return (a, b) => {
-      if (!parseInt(a) && !parseInt(b)) {
+      if (!Number(a) && !Number(b)) {
```

Comment:
should we test using `isNaN`?

```js
isNaN(Number(a)) && isNaN(Number(b))
```

Reply:
`Number` returns `NaN` so should be covered here

Reply:
`isNaN` distinguishes between `0` and `NaN`. If `0` is to be treated the same, then what you have is fine.